paper.md

```diff
@@ -167,15 +167,15 @@ Established and widely-adopted infrastructure and procedures, like software dist
 However, data have remained a “2nd-class citizen” in the contemporary scientific process, despite the increasing embracement of the FAIR principles [@FAIR2016].
 1) Disconnected data portals force a cacophony of data access and authentication methods upon the scientific community; 2) data versioning is rarely performed, or nowhere close to the precision and inspectability that is standard in software development; and 3) data provenance---if shared at all---is often not complete, because their capture is rarely considered to be an integral part of the scientific process.
 DataLad aims to solve all these issues by providing targeted interfaces and interoperability adapters to standard tools and services, scientific and commercial.
-It enables workflows that are particularly suited for reproducible science, such as actionable process provenance capture for arbitrary command execution that affords automatic re-execution.
+It enables workflows that are particularly suited for reproducible science, such as actionable process provenance capture for arbitrary command execution that enables automatic re-execution.
 In order to maximize its utility and target audience, DataLad is available for all major operating systems, and can be integrated into established workflows and environments with minimal adjustments.
 
 ## Why Git and git-annex?
 
 Git is the most popular version control system for software development[^1].
 It is a distributed content management system, specifically tuned towards managing and collaborating on text files, and excels at making all content committed to Git reliably and efficiently available to all clones of a repository.
 At the same time, Git is not designed to efficiently handle large (e.g., over a gigabyte) or binary files [see, e.g., @opensource:git-binary].
-This makes it hard or impossible to use Git directly for distributed data storage with tailored access to individual files, or even to fully remove file content without severely impacting the integrity of a repository.
+This makes it hard or impossible to use Git directly for distributed data storage with tailored access to individual files. <!-- IMHO the following sentence is unnecessarilly detailed and ultimately confusing for non Git-cracks, so I removed it: ", or even to fully remove file content without severely impacting the integrity of a repository." -->
```

Comment:
I'd disagree here. It may not be clear why this is a problem in Git, but it is an important point that removing content from a repository is a problem in (pure) Git.

Reply:
I agree that it is a technical problem to remove content in pure Git. But I think we could subsume it under "it [is] hard or impossible to use Git directly for distributed data storage". It seems unnecessarily detailed for a reader that is not familiar with git intricases.

Reply:
I am OK to remove it. I wrote it, because I was trying to keep the original list of points before my edit.